cmd/cert-checker/main.go

```diff
@@ -122,12 +131,22 @@ func (c *certChecker) getCerts(unexpiredOnly bool) error {
 		args["now"] = c.clock.Now()
 	}
 
-	sni, err := c.dbMap.SelectNullInt(
-		"SELECT MIN(id) FROM certificates WHERE issued >= :issued AND expires >= :now",
-		args,
-	)
-	if err != nil {
-		return err
+	var sni sql.NullInt64
+	var err error
+	var backoff int
+	for {
+		sni, err = c.dbMap.SelectNullInt(
+			"SELECT MIN(id) FROM certificates WHERE issued >= :issued AND expires >= :now",
+			args,
+		)
+		if err != nil {
+			c.logger.AuditErrf("finding starting certificate: %s", err)
+			backoff++
+			time.Sleep(time.Second)
```

Comment:
Here you always sleep for 1 second, while below you sleep for `backoff` number of seconds.

I think both of these would be great places to use `core.RetryBackoff()` to avoid reinventing the wheel and ensure we get it right.